### DISCUSSION TITLE
Icloud Albums & Folders

### DISCUSSION THREAD

**Discussion by JyFu-py** [steilerDev/icloud-photos-sync#151] [Q&A] [answered]
Hello,

Is there a way to reproduce icloud photo Albums & Folders when syncing ?

![image](https://user-images.githubusercontent.com/67822884/211214869-e70f4b00-6cb7-4c08-a7a4-ec544f5e4ac9.png)

Thanks !

**Comment by steilerDev** [comment]
Yeah, the result will be something like this

**Comment by JyFu-py** [comment]
Perfect !
When is the tree structure created?
I only have files in _All-Photos at the moment.

Thanks !

**Comment by steilerDev** [comment, marked as answer]
Albums are synced once all Assets have been synced correctly (as I'm only linking the files from '_All Photos' into the correct folders)

So we first need to get the initial Sync working for you :D
